lib/tasks/secretsmanager.rake

```diff
@@ -49,4 +49,42 @@ namespace :secretsmanager do
       exit 1
     end
   end
+
+  desc "Autogenerate secret_key_base for Rails apps"
+  task :autogenerate_secret_key_base do
+    config = JSON.parse(File.read("../terraform-outputs/secret_key_bases.json"))
+
+    credentials = Aws::AssumeRoleCredentials.new(
+      client: Aws::STS::Client.new,
+      role_arn: ENV["ASSUME_ROLE_ARN"],
+      role_session_name: "create-rails-app-secret_key_base",
+    )
+    secrets_client = Aws::SecretsManager::Client.new(
+      region: ENV["AWS_REGION"],
+      credentials: credentials,
+    )
+
+    config.fetch("arns").each do |secret_arn|
+      metadata = secrets_client.describe_secret(secret_id: secret_arn)
+      versions = metadata.version_ids_to_stages
+      is_unset = versions.nil? || versions.values.none? do |stages|
+        stages.include?("AWSCURRENT")
+      end
+
+      unless is_unset
```

Comment:
For clarity I'd consider avoiding the double-negative here. i.e. I'd reverse the sense of this bool, call it `is_set` (or perhaps `already_set`) and change the unless to an if. (There's a great [testing on the toilet](https://testing.googleblog.com/2007/01/introducing-testing-on-toilet.html) episode for this but I can't find it. It's similar to [this blog post](https://derickbailey.com/2014/03/27/double-negatives-in-code-dont-not-avoid-double-negative-boolean-logic/) though.)